docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:4.2.4-wheezy
+MAINTAINER Rodney Gomes (rodney@jut.io)
+
+CMD outriggerd
+
+RUN npm install outrigger juttle -g
```

Comment:
This is a total nit but I've more often seen this invoked as `npm install -g ...`

Also I think we should include several (all?) of the adapters. It's basically zero cost since they don't get loaded unless the user specifies it in the config.